grin/app/GrinCLI.hs

```diff
@@ -40,9 +41,14 @@ transformOpts =
   <|> flg UpdateElimination "ue" "Update Elimination"
   <|> flg CopyPropagation "cp" "Copy Propagation"
   <|> flg ConstantPropagation "cnp" "Constant Propagation"
-  <|> flg DeadProcedureElimination "dpe" "Dead Procedure Elimination"
-  <|> flg DeadParameterElimination "dae" "Dead Parameter Elimination"
+  <|> flg DeadCodeElimination "dce" "Dead Code Elimination"
+  <|> flg DeadDataElimination "dde" "Dead Data Elimination"
+  <|> flg DeadFunctionElimination "dfe" "Dead Function Elimination"
+  <|> flg DeadParameterElimination "dpe" "Dead Parameter Elimination"
   <|> flg DeadVariableElimination "dve" "Dead Variable Elimination"
+  <|> flg SimpleDeadFunctionElimination "sdfe" "Dead Procedure Elimination"
```

Comment:
Is *dead procedure elimination* or *dead function elimination* the better naming?
I prefer the Boq PhD naming, because it's our documentation so far.

Reply:
Dead Parameter Elimination's and Dead Procedure Elimination's abbreviation would clash.